### PR TITLE
Fix t2s in Text

### PR DESCRIPTION
**Pull request by naveen521kk**
## List of Changes
- Fix t2s in `Text`.

## Explanation for Changes
Say the changes in cairo context to pango layout when we actually move
using pangocairocffi's update_layout function simply fix this
Resolves https://github.com/ManimCommunity/manim/issues/636

## Testing Status
Try locally and it works. I will add a small test soon.

## Acknowledgement
- [x] I have read the [Contributing Guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP))

### CODE REVIEW

File: tests/test_pango.py [hunk 1]
```diff
@@ -137,3 +137,20 @@ def test_tabs_replace() -> None:
...
+    context.move_to(START_X, START_Y)
+    layout = pangocairocffi.create_layout(context)
+    layout.set_width(pangocffi.units_from_double(WIDTH))
+    fontdesc = pangocffi.FontDescription()
+    fontdesc.set_size(pangocffi.units_from_double(size * 10))
+    layout.set_font_description(fontdesc)
+    layout.set_markup('Hello<span style="italic">world</span>')  # yay, pango markup
+    pangocairocffi.show_layout(context, layout)
+    surface.finish()
+    assert compare_SVGObject_with_PangoText(temp_pango_text, filename)
```
Comment (leotrs) [inline tests/test_pango.py:156]: Comment (non-blocking): in the future, we should refactor this code into a helper function (it is mostly repeated from other tests).

**Review by leotrs — APPROVED** [review]

File: tests/test_pango.py (hunk 1 above)
Comment (huguesdevimeux) [reply tests/test_pango.py:156]: We should also rename all of the t2thing stuff

Comment (naveen521kk) [reply tests/test_pango.py:156]: Not for now. Maybe open a new issue @huguesdevimeux ?

Comment (huguesdevimeux) [reply tests/test_pango.py:156]: yes,of course (it was I was working on) (but won't finish before long)

File: tests/test_pango.py [hunk 2]
```diff
@@ -137,3 +137,20 @@ def test_tabs_replace() -> None:
     pangocairocffi.show_layout(context, layout)
     surface.finish()
     assert compare_SVGObject_with_PangoText(temp_pango_text, filename)
+
+
+def check_t2s() -> None:
```
Comment (huguesdevimeux) [inline tests/test_pango.py:142]: **Question** i'm pretty sure it's intended, but I can't see the reason. Why is it `check_t2s` and not `test_t2s`?

**Review by huguesdevimeux — APPROVED** [review]
SGTM

File: tests/test_pango.py (hunk 2 above)
Comment (naveen521kk) [reply tests/test_pango.py:142]: That needs to be `test`.

**Pull request merged**